apps/rush-lib/src/cli/actions/BaseRushAction.ts

```diff
@@ -11,19 +12,43 @@ import {
 import RushConfiguration from '../../data/RushConfiguration';
 import EventHooksManager from '../logic/EventHooksManager';
 
+export interface IRushCommandLineActionOptions extends ICommandLineActionOptions {
+  /** If true, no locking mechanism will be enforced when this action is run */
```

Comment:
>/** If true, no locking mechanism will be enforced when this action is run */ [](start = 2, length = 77)

Does this default to false? Can you note that?

Reply:
I found "(which is safer)" a little confusing, since the option marks things as being safe.

---
In reply to: [165489554](https://github.com/Microsoft/web-build-tools/pull/517#discussion_r165489554) [](ancestors = 165489554)